### DISCUSSION TITLE
Disabling httpx on massive target list.

### DISCUSSION THREAD

**Discussion by justgu3st** [projectdiscovery/nuclei#3830] [Q&A]
As you guys know when we run command like;
~root@ nuclei -l targets.txt -t /nuclei-templatest/takeovers/  it is going through httpx and nucelei will start to scan targets before POST requests. In my case i got millions of subdomains and i do not want to scan all these targets because it is loosing time.
Is it any way to disable that requests without changing source code of nuclei ?

**Comment by ehsandeep** [comment]
@justgu3st, you can disable httpx probing with the below option, but since you are running the takeover template, you need either probe subdomain in advance or nuclei will probe for you.

```console
   -nh, -no-httpx  disable httpx probing for non-url input
```

**Reply by justgu3st** [reply]
Will it work properly if i use "-nh" on takeovers template ?

**Reply by ehsandeep** [reply]
It will not at all, as you are giving subdomains as input; takeover templates requires URLs as input to run.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
